tools/compression_v2_stats.py

```diff
@@ -0,0 +1,77 @@
+"""
+This file is a plug-in to the `analyze-chain` tool.
+
+To use, run with a command-line like:
+
+```
+DB=~/.chia/mainnet/db/blockchain_v2_mainnet.sqlite
+python -m tools.analyze-chain $DB --start 300000 --end 300100  --call tools.compression_v2_stats:stats
+```
+
+while the current directory is the root `chia-blockchain` folder.
+
+It will iterate over transaction blocks, recompress them and print the new cost ("agony rating")
+along with a relative percentage, where 100% is par and smaller is better.
+"""
+
+import time
+from typing import List
+
+from chia_rs import compression  # type: ignore[attr-defined]
```

Comment:
is this warning about `chia_rs` being imported as well, somewhere?

Reply:
The wheel I installed didn't have updated mypy headers. I'll have to figure out how to do that.

Reply:
```console
$ venv/bin/python -c 'from chia_rs import compression'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name 'compression' from 'chia_rs' (/home/altendky/repos/chia-blockchain/venv/lib/python3.10/site-packages/chia_rs/__init__.py)
```

Reply:
Correct, this is a feature preview. The compression module only exists in a branch of `chia_rs` at the moment.